boards/weact/blackpill_u585ci/doc/index.rst

```diff
@@ -0,0 +1,209 @@
+********
+
+The WeAct Black Pill STM32U585CI Core Board is an extremely low cost and bare-bone
+development board featuring the STM32U585CI, see `STM32U585CI website`_.
+This is the 48-pin variant of the STM32U585C series,
+see `STM32U5 reference manual`_. More info about the board available
+on `WeAct Github`_.
+
+Key Features
+
+- STM32U585CI microcontroller in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg| -M33 with TrustZone |reg|, MPU, DSP and FPU
+- 160 MHz max CPU frequency, 1.5 DMPIS/MHz, 4.07 CoreMark |reg| /MHz
+- 2 MB Flash with ECC and 2 banks
+- 784 KB SRAM
+- GPIO with external interrupt capability
+- 1x14-bit 2.5 MSPS ADC, 1x12-bit 2.5 MSPS ADC
+- 2x12-bit DAC
+- USB OTG 2.0 full-speed controller
+- 1 user LED
+- User, boot, and reset push-buttons
+- 32.768 kHz and 8MHz HSE crystal oscillators
+- Board connectors:
+
+   - USB Type-C |reg| Connector
+   - SWD header for external debugger
+   - 2x 20-pin GPIO connector
+
+Hardware
+********
+
+The STM32U585xx devices are an ultra-low-power microcontrollers family (STM32U5
+Series) based on the high-performance Arm |reg| Cortex |reg| -M33 32-bit RISC core.
+They operate at a frequency of up to 160 MHz.
+
+- Ultra-low-power with FlexPowerControl (down to 210 nA Standby mode and 19.5 uA/MHz run mode)
+
+- Arm |reg| TrustZone |reg| and securable I/Os memories and peripherals
+- Secure Firmware Update support with TF-M
+- HASH hardware accelerator
+- True Random Number Generator NIST SP800-90B compliant
+- 96-bit unique ID
+- 512-byte One-Time Programmable for user data
+
+- 4 to 50 MHz crystal oscillator
+- 32 kHz crystal oscillator for RTC (LSE)
+- Internal 16 MHz factory-trimmed RC ( |plusminus| 1%)
+- Internal low-power 32 kHz RC ( |plusminus| 5%)
+- 2 internal multispeed 100 kHz to 48 MHz oscillators, including one auto-trimmed by
+  LSE (better than |plusminus| 0.25 % accuracy)
+- 3 PLLs for system clock, USB, audio, ADC
+- Internal 48 MHz with clock recovery
+
+- Up to 136 fast I/Os, most 5 V-tolerant, up to 14 I/Os with independent supply down to 1.08 V
+- Up to 22 capacitive sensing channels: support touchkey, linear and rotary touch sensors
+- Up to 17 timers and 2 watchdogs
+
+- USB Type-C |reg| / USB power delivery controller
+- USB OTG 2.0 full-speed controller
+- 2x SAIs (serial audio interface)
+- 4x I2C FM+(1 Mbit/s), SMBus/PMBus
+- 6x USARTs (SPI, ISO 7816, LIN, IrDA, modem)
+- 3x SPIs (+2 SPIs with dual OCTOSPI, +3 with USART)
+- 1x FDCAN
+- 2x SDMMC interface
+- 16- and 4-channel DMA controllers, functional in Stop mode
+- 1 multi-function digital filter (6 filters)+ 1 audio digital filter with
+  sound-activity detection
+
+- CRC calculation unit
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+- True Random Number Generator (RNG)
+- Chrom-ART Accelerator (DMA2D) for enhanced graphic content creation
+- 1 digital camera interface
+- CORDIC for trigonometric functions acceleration
+- FMAC (filter mathematical accelerator)
```

Comment:
there is no need for the documentation page of this board to be a commercial brochure for the STM32U585xx SoC series. This is taking real estate on the documentation page of the board for no good reason as referencing the website and reference manual should be more than enough.

```suggestion
```